gravitee-apim-console-webui/src/scss/gio-global-style.scss

```diff
@@ -8,26 +8,42 @@
 
 $background: map.get(gio.$mat-theme, background);
 
-.gio-snack-bar-success {
-  white-space: pre-wrap;
-  background-color: mat.get-color-from-palette(gio.$mat-success-palette, default);
-  color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
-
-  .mat-button {
+.gio-snack-bar-success.mat-mdc-snack-bar-container {
+  .mdc-snackbar__surface {
+    white-space: pre-wrap;
+    background-color: mat.get-color-from-palette(gio.$mat-success-palette, default);
     color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
+
+    .mat-mdc-button {
+      color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
+    }
+
+    .mat-mdc-snack-bar-label.mdc-snackbar__label {
+      color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
+    }
   }
 }
 
-.gio-snack-bar-error {
-  white-space: pre-wrap;
-  background-color: mat.get-color-from-palette(gio.$mat-error-palette, default);
-  color: mat.get-color-from-palette(gio.$mat-error-palette, default-contrast);
-
-  .mat-button {
+.gio-snack-bar-error.mat-mdc-snack-bar-container {
+  .mdc-snackbar__surface {
+    white-space: pre-wrap !important;
+    background-color: mat.get-color-from-palette(gio.$mat-error-palette, default);
```

Comment:
It may be more interesting to control the background color globally, like here: https://stackoverflow.com/a/74704599

Reply:
Done